driver_guided_robot/include/control_system/system_manager.hpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Zoltán Rési
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CONTROL_SYSTEM__SYSTEM_MANAGER_HPP_
+#define CONTROL_SYSTEM__SYSTEM_MANAGER_HPP_
+
+#include <vector>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
+#include "lifecycle_msgs/srv/change_state.hpp"
+#include "lifecycle_msgs/msg/state.hpp"
+#include "sensor_msgs/msg/joint_state.hpp"
+#include "std_msgs/msg/bool.hpp"
+#include "std_srvs/srv/set_bool.hpp"
+#include "std_srvs/srv/trigger.hpp"
+#include "kuka_sunrise_interfaces/srv/get_state.hpp"
+#include "kuka_sunrise/internal/service_tools.hpp"
+
+namespace driver_guided_robot
+{
+
+// TODO(kovacsge11) maybe derive from ROS2BaseNode
```

Comment:
Writing this comment so that you notice this TODO, this is a possible not so high prio TODO.